src/main/java/us/tastybento/bskyblock/api/flags/FlagBuilder.java

```diff
@@ -1,19 +1,17 @@
 package us.tastybento.bskyblock.api.flags;
 
-import java.util.Optional;
-
 import org.bukkit.Material;
 import org.bukkit.event.Listener;
-
 import org.bukkit.inventory.ItemStack;
+
 import us.tastybento.bskyblock.api.panels.PanelItem;
 import us.tastybento.bskyblock.api.panels.builders.PanelItemBuilder;
 
 public class FlagBuilder {
 
     private String id = "";
-    private PanelItem icon = PanelItem.empty();
-    private Optional<Listener> listener = Optional.empty();
+    private PanelItem icon;
```

Comment:
I didn't mean to remove this - I was experimenting with the testing and ItemStack has static methods in it that disallow mocking. It's fine to keep this in now because I gave up trying to get static method to mock (2 hours wasted!)